roles/confluent.test/upgrade_testing/zookeeper-tls-upgrade-620-current.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+## Variables
+
+export SCENARIO_NAME=zookeeper-tls-rhel
+export START_BRANCH=6.1.x
+export START_UPGRADE_VERSION=6.1
+export CURRENT_VERSION=true
+
+#Add tags to select specific component installation.
+export TAG_NAME="-- --tags zookeeper"
+
+
+## If current version is set to true, will change END_BRANCH to be equal to the latest CP BRANCH.
+if [[ $CURRENT_VERSION == true ]]
+then
+  export END_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+fi
+
+## Change to project root
+cd ..
+
+## Checkout starting branch
+echo "Checking out $START_BRANCH branch"
+git checkout $START_BRANCH
+
+## Run Molecule Converge on scenario
+echo "Running molecule converge on $SCENARIO_NAME with tags $TAG_NAME"
+molecule converge -s $SCENARIO_NAME $TAG_NAME
+
```

Comment:
Should we not run the verify.yml at pre and post upgrade state?

Reply:
We do not run verification after upgrades as we do not modify configs in any way, we simply upgrade the binaries.  If the upgrades fail to work the health checks will fail on the given component.